### PR TITLE
add bulkinsert options

### PR DESCRIPTION
**Pull request by jaime0815**

### CODE REVIEW

File: client/options.go [hunk 1]
```diff
@@ -247,6 +248,110 @@ func IsBackup() BulkInsertOption {
...
+		optionMap := entity.KvPairsMap(req.GetOptions())
+		optionMap["access_kye_id"] = accessKeyID
+		req.Options = entity.MapKvPairs(optionMap)
+	}
+}
+
+func WithSecretAccessKeyID(secretAccessKeyID string) BulkInsertOption {
+	return func(req *milvuspb.ImportRequest) {
+		optionMap := entity.KvPairsMap(req.GetOptions())
+		optionMap["secret_access_key_id"] = secretAccessKeyID
```
Comment (czs007) [inline client/options.go:286]: I suggest aligning the parameter names here with the parameter names in milvus.yaml, except for start_ts and end_ts. These two parameters（start_ts, end_ts） are only used by the backup tool, so it is acceptable to have inconsistent naming styles.

image

File: client/options.go [hunk 2]
```diff
@@ -247,6 +248,110 @@ func IsBackup() BulkInsertOption {
 	}
 }
 
+func WithBucket(bucket string) BulkInsertOption {
+	return func(req *milvuspb.ImportRequest) {
+		optionMap := entity.KvPairsMap(req.GetOptions())
+		optionMap["bucket"] = bucket
```
Comment (czs007) [inline client/options.go:254]: bucketName

Comment (czs007) [reply client/options.go:254]: params.add("address", "localhost")
params.add("port", 9000)
params.add("accessKeyID", "minioadmin")
params.add("secretAccessKey", "minioadmin")
params.add("useSSL", False)
params.add("bucketName", "a-bucket")
params.add("rootPath", "files")
params.add("useIAM", False)
params.add("cloudProvider", "aliyun")
params.add("iamEndpoint", "")
params.add("region", "")
params.add("useVirtualHost", True)

Comment (jaime0815) [reply client/options.go:254]: bucket is an already existing name, it can't be compatible with old version if we change it

**Pull request closed without merging**